test/jdk/java/time/tck/java/time/TCKZoneId.java

```diff
@@ -96,6 +96,7 @@ public class TCKZoneId extends AbstractTCKTest {
     //-----------------------------------------------------------------------
     // SHORT_IDS
     //-----------------------------------------------------------------------
+    @Test
```

Comment:
A passing comment is that I personally find it confusing to see `@Test` on the class and the test methods. With TestNG then I think you can use one or the other. It looks like the intention here is to put the annotation on the methods and eventually migrate to JUnit, is that right?